scripts/buildocaml.sh

```diff
@@ -2,25 +2,30 @@
 set -e
 
 
-export OCAMLPARAM='_,bin-annot=1' 
+export OCAMLPARAM='_,bin-annot=1'
 export OCAMLRUNPARAM=b
 
 if [ $BS_TRAVIS_CI ]
-then 
+then
     git submodule update --init --recursive
     OCAML=ocaml
-else 
+else
     OCAML=ocaml_src
     if [  -f "ocaml.tar.gz" ]
-    then 
+    then
         echo "ocaml.tar.gz already exists"
         rm -rf $OCAML
         mkdir -p $OCAML
         tar -zxvf ./ocaml.tar.gz -C $OCAML
     else
         echo "creating ocaml from github"
-        git clone -b master --depth 1 https://github.com/bloomberg/ocaml $OCAML
+        if [ -d "$OCAML" ]
+        then
```

Comment:
Correct. That is what this would fix. I am currently running into this with [`reductive`](https://github.com/reasonml/reductive), but previously when using bucklescript from GitHub instead of npm. Also was hitting it with `reason-react` before they switched to `bucklescript@1.6.0` in npm.

I'm going to submit a PR over to `reductive` as well to track the npm release instead of master here, but I still think this PR will be a good thing to have.